domain/src/test/java/org/oppia/android/domain/classify/rules/fractioninput/FractionInputIsEquivalentToAndInSimplestFormRuleClassifierProviderTest.kt

```diff
@@ -24,15 +24,13 @@ class FractionInputIsEquivalentToAndInSimplestFormRuleClassifierProviderTest {
 
   private val WHOLE_NUMBER_VALUE_TEST_5 =
     InteractionObjectTestBuilder.createWholeNumber(
-      isNegative = false,
       value = 5
     )
   private val WHOLE_NUMBER_VALUE_TEST_2 =
     InteractionObjectTestBuilder.createWholeNumber(
-      isNegative = false,
       value = 2
     )
-  private val NEGATIVE_WHOLE_NUMBER_VALUE_TEST_2 =
+  private val NEGATIVE_INTEGER_VALUE_TEST_2 =
     InteractionObjectTestBuilder.createWholeNumber(
       isNegative = true,
```

Comment:
Could you help in understanding here why we are creating a negative whole number here?
As this is wrong and this is something we are solving in this issue.

Reply:
I have updated this function call to createMixedNumber.